package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "@khanacademy/eslint-plugin",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "publishConfig": {
     "access": "public"
   },
   "files": [
-    "/lib"
+    "/lib",
+    "/node_modules/checksync/bin",
```

Comment:
Err wait... this doesn't make sense. You're shipping checksync as part of the package? Why? checksync is already a dependency, we should be using that dependency! This is definitely wrong.